torch/utils/module_tracker.py

```diff
@@ -74,6 +93,54 @@ def is_bw(self):
         """
         return torch._C._current_graph_task_id() != -1
 
+    def get_known_fqn(self, mod):
+        """
+        Return the fqn for the given module if it is known to the Module Tracker, otherwise None
+        """
+
+        if mod not in self._known_modules:
+            return None
+        return self._known_modules[mod]
+
+    def register_user_hooks(
+        self,
+        pre_fw_hook: Optional[Callable] = None,
+        post_fw_hook: Optional[Callable] = None,
+        pre_bw_hook: Optional[Callable] = None,
+        post_bw_hook: Optional[Callable] = None,
+    ):
+        """
+        Registers user specified hooks that are called before/after the forward/backward pass for
```

Comment:
Can you update the styling of the doc here to match what is expected by our doc style. An example is here: https://github.com/pytorch/pytorch/blob/cc4da72b47ef63b7c448f0de4cdbdd792e9195ea/torch/_torch_docs.py#L336